### DISCUSSION TITLE
Automations in nocodb [Aug 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6248] [Discord Threads]
> 🙋‍♂️ **knicklicht4421**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### you can use webhooks for each table row create/update/delete  
#  
> 🙋‍♂️ **knicklicht4421**  
#### Thanks for the response. I am trying to achieve the following: I have two tables A and B and each time there is a change in A I want to create a row in B. I could write my own code deploy it etc. based on web hooks. But I was wondering if there is essentially a drag and drop kind of interface to define such automations inside NocoDB.  
#  
> 💁 **o1lab**  
#### Nope, any other tool as a reference that allows u ve in mind  
#  
> 🙋‍♂️ **knicklicht4421**  
#### Does the NocoDB Team suggest any 3rd party tools that work particularly well with NocoDB?  
#  
> 💁 **o1lab**  
#### Any automation tool is fine  
#  
> 🙋‍♂️ **knicklicht4421**  
#### Okay, thanks  
#  
> 🙋‍♂️ **CL**  
#### I think you can check out n8n, it is quite a nice one and easy to deploy  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1130853542773673995)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by felipemeres** [comment]
I use n8n with nocodb and highly recommend it
